src/library/librarycontrol.cpp

```diff
@@ -292,8 +303,19 @@ void LibraryControl::slotSelectTrack(double v) {
     activeView->moveSelection(i);
 }
 
+void LibraryControl::slotSelectItem(double v) {
+    if (m_pLibraryWidget == nullptr || m_pSidebarWidget == nullptr) {
+        return;
+    }
+    // Select track if a LibraryView object has focus, otherwise select sidebar item
+    if (m_pLibraryWidget->getActiveView()->hasFocus()) {
```

Comment:
getActiveView can return null so we should check that the result is not null before dereferencing